disaster-recovery.md

```diff
@@ -0,0 +1,168 @@
+   
+### Scenario 2.4: If the incorrect configuration CANNOT be changed via governance and the parameter IS part of the checkpoint data 
+
+1. Stop the network in coordination with all validators and inform Vega team immediately
+2. Vega team to create a software release that would hardcode the value in question into the binary to resolve the issue.
+3. Validators would need to ALL deploy the latest code
+4. Restart the network using the last [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+
+**NOTE:** Hardcoding values would be implemented at a block height so that it’s only hard coded for the time it’s needed, not in the software until the next deployment.
+
+
+## Scenario 3: Less than 2/3+1 of the validators are active
+
+### Scenario 3.1: If the network has not been running long
+
+1. Restart the node(s) and see the chain data. 
+2. Each node will try to replay the chain to the latest block. 
+
+**NOTE:** As chain events grow, the data required to catch up will become too large for this to be a viable action.
+
+### Scenario 3.2: If 3.1 is not viable due to age of the network
+
+1. Shut down the full network with coordination between validators.
+2. Restart the network from the last [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+
+**NOTE:** If there is a planned outage happening soon the node(s) could be brought back at that time to avoid unplanned outages.
+
+## Scenario 4: Critical bugs
+
+The following section describes how to resolve critical bugs that manifest in various components within the Vega application stack.
+
+### Scenario 4.1: Critical bug in Vega core ABCI application
+
+In the event that a critical bug is found in the protocol software that falls under one of the following conditions that means the network MUST be stopped:
+* Severe financial loss
+* The network is not operational
+* Someone can change balances
+
+If this should occur the following actions should be taken
+
+1. Shut down the full network with coordination between validators.
+2. The Vega team will replay the chain to investigate the issue
+3. The Vega team will create a new software release that fixes the bug
+4. Validators would ALL need to deploy the latest software release (assuming sucessful governance vote to deploy)
+5. Restart network using the latest [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint). 
+
+
+### Scenario 4.2: Critical big in a smart contract used by the network
+
+In the event that a critical bug is found in tone or more of the Smart Contracts that falls under one of the following conditions::
+* Severe financial loss
+
+If this should occur the following actions should be taken:
+
+1. Shut down the full network with coordination between validators.
+2. The Vega team will investigate the issue
+3. The Vega team will create a new smart contract release that fixes the bug
+4. Restart the network using the latest checkpoint file
```

Comment:
@JonRay15 @MM0819 - should we update this to:

4. Validators would ALL need to update the config to point to the new contract address (assuming successful governance vote to carry out this action)
5. Restart network using the latest [checkpoint](https://github.com/vegaprotocol/networks#restore-from-checkpoint).

Reply:
UPdated in line with comments